crawler/constants.py

```diff
@@ -119,6 +120,12 @@
 # DART others
 DART_SET_PROP_STATUS_SUCCESS = 0
 
+# If we're running in a container, then instead of localhost
```

Comment:
I don't think we should be putting connection config in the constants file. You could just move `os.environ.get("LOCALHOST", "127.0.0.1")` to the default config file. Test imports all config from defaults.

Reply:
What's your opinion on the ROOT_PASSWORD one, @pjvv ? same deal?

Reply:
Oh yeah, same, sorry was just being lazy and used that one as an example!